Twitter/Repositories/FastPostRepository.cs

```diff
@@ -29,6 +30,18 @@ public async Task<FastPost> CreateFastPost(FastPost postRequest)
 			return postRequest;
 		}
 
+		public async Task<List<FastPost>> FindAllFastPostsAsync()
+		{
+			var fastPosts = await FindByCondition(fastPost => EF.Functions.DateDiffHour(fastPost.Created, DateTime.Now) < 24).ToListAsync();
+
+			if (fastPosts.Count == 0)
+			{
+				throw new FastPostNotFoundException("FastPosts are no longer available");
```

Comment:
It's also ok to return an empty string. You don't need to throw exceptions if you don't have fastPosts. This is not the case when you are looking for exactly one post.

Reply:
I've fixed it.